app/src/main/java/com/futurice/freesound/network/api/model/mapping/DateAdapter.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+package com.futurice.freesound.network.api.model.mapping
+
+import com.squareup.moshi.FromJson
+import java.util.*
+
+class DateAdapter {
+
+    @FromJson fun fromJson(json: String): Date {
+        val parsableDate = if (json.endsWith("Z")) json else json.plus("Z")
```

Comment:
Because the date format that freesound provides is not according to the standard. The `Z` from the end is missing